.github/workflows/release.yml

```diff
@@ -381,6 +383,11 @@ jobs:
       run: |
         tar -xf source.tar.gz --strip-components=1
 
+    - name: Update vcpkg
+      run: |
+        cd /usr/local/share/vcpkg
+        git pull
```

Comment:
As a FYI, I think that the `doNotUpdateVcpkg` just a little bit lower does exactly this. But I am not 100% sure :)

Reply:
When I checked the source code for that action, the only reference I can find to that phrase is in the documentation. I can't find any actual code behind it. If this does work, I will try out the doNotUpdateVcpkg too.

Eventually I expect the version of vcpkg on the build server will be updated and this will be somewhat redundant (by updating every time it will prevent caching I imagine, so we will perhaps need to checkout a specific revision).